src/pages/buy-sell/screens/BuySellHeader/BuySellHeader.tsx

```diff
@@ -43,6 +46,16 @@ const BuySellHeader = ({
     const { localize } = useTranslations();
     const { isMobile } = useDevice();
     const isAdvertiserBarred = useIsAdvertiserBarred();
+    const { setQueryString } = useQueryString();
+
+    api.advert.useGetList({
```

Comment:
why are we calling this here, since we are already passing the setters to the parent.